conans/test/integration/cache/backup_sources_test.py

```diff
@@ -154,6 +156,7 @@ def source(self):
             path=self.client.cache.cache_folder)
         self.client.run("create .")
         self.client.run("upload * -c -r=default")
+        assert "FileNotFoundError: [Errno 2] No such file or directory" not in self.client.out
```

Comment:
This might be tricky. It is possible that with other Pythons, other OSs, the error message is not the same. I'd try to ``assert "error" not in str(self.client.out).lower()`` or something like that.